llvm/tools/sycl-post-link/SYCLDeviceLibReqMask.cpp

```diff
@@ -657,16 +657,16 @@ SYCLDeviceLibFuncMap SDLMap = {
 // is an unsigned int32. getDeviceLibBit checks which fallback device library
 // is required for FuncName and returns the corresponding bit. The corresponding
 // mask for each fallback device library is:
-// fallback-cassert:      0x1
-// fallback-cmath:        0x2
-// fallback-cmath-fp64:   0x4
-// fallback-complex:      0x8
-// fallback-complex-fp64: 0x10
-// fallback-cstring:      0x20
-// fallback-imf:          0x40
-// fallback-imf-fp64:     0x80
-// fallback-imf-bf16:     0x100
-// fallback-bfloat16:     0x200
+// cl_intel_devicelib_assert:        0x1
```

Comment:
I am a bit concerned that none of these values are present in SDLMap above.

Reply:
> I am a bit concerned that none of these values are present in SDLMap above. @jinge90 Any comments on that?

Hi, @maksimsab 
Which "values" do you refer to?
Thanks very much.

Reply:
Sorry, I've found it.